jsk_fetch_robot/jsk_fetch_startup/launch/fetch_bringup.launch

```diff
@@ -192,6 +193,25 @@
       <arg name="launch_map_server" value="false" />
       <arg name="odom_topic" value="/odom_combined" />
     </include>
+    <!-- relay plan path topic for visualization -->
```

Comment:
`/move_base/<global planner>/plan` and `/move_base/<local planner>/global_plan` looks almost same except replanning of global plan. What is the purpose of splitting their visulization?
cf. https://wiki.ros.org/base_local_planner#API_Stability

Reply:
@sktometometo 

> looks almost same

Yes, I think so, too.

The only reason I publish both `/move_base/<global planner>/plan` and `/move_base/<local planner>/global_plan` is that they have been visible in RViz config in fetch for a long time.

I will create another PR to reduce visualization.
Which do you think should be visualized, `/move_base/<global planner>/plan` or `/move_base/<local planner>/global_plan`?